contracts/mocks/compound/CompTimelock.sol

```diff
@@ -31,23 +31,23 @@ contract CompTimelock {
     event NewPendingAdmin(address indexed newPendingAdmin);
     event NewDelay(uint256 indexed newDelay);
     event CancelTransaction(
-        bytes32 indexed txHash,
+        bytes32 indexed operationHash,
```

Comment:
Sorry for comming back to it, but actuelly this is a contract we vendor from Coumpound. Its in the mocks because we only use it for tests, but it should probably be in `contracts/vendor/compound`. 

I'm not sure we should touch these vendored contract at all.

Reply:
The difference between this and `contracts/vendor/compound` is that this doesn't end up in the npm package.

Reply:
@frangio WDYT ?

Reply:
Agree the vendored contracts shouldn't be changed. They don't need to follow our conventions. In fact we could ignore them from linter rules for example.

Reply:
I made the change just for consistency without further context on this contract and I initially considered it better to be consistent with the named mapping arguments but I agree this is not required.

I reverted back to the original version. Solving.